src/bin/vaults/CommandStat.ts

```diff
@@ -1,98 +0,0 @@
-// Import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
-// import PolykeyClient from '../../PolykeyClient';
-// import * as vaultsPB from '../../proto/js/polykey/v1/vaults/vaults_pb';
-// import * as utils from '../../utils';
-// import * as binUtils from '../utils';
-
-// import * as grpcErrors from '../../grpc/errors';
-
-// import CommandPolykey from '../CommandPolykey';
-// import * as binOptions from '../utils/options';
-
-// class CommandStat extends CommandPolykey {
-//   constructor(...args: ConstructorParameters<typeof CommandPolykey>) {
-//     super(...args);
-//     this.name('stat');
-//     this.description('Vaults Stat');
-//     this.requiredOption(
-//       '-vn, --vault-name <vaultName>',
-//       '(required) Name of the vault to get stats from',
-//     );
-//     this.addOption(binOptions.nodeId);
-//     this.addOption(binOptions.clientHost);
-//     this.addOption(binOptions.clientPort);
-//     this.action(async (options) => {
-
-//     });
-//   }
-// }
-
-// export default CommandStat;
-
-// OLD COMMAND
-// const stat = binUtils.createCommand('stat', {
-//   description: 'Gets stats of an existing vault',
-//   nodePath: true,
-//   verbose: true,
-//   format: true,
-// });
-// stat.requiredOption(
-//   '-vn, --vault-name <vaultName>',
-//   '(required) Name of the vault to get stats from',
-// );
-// stat.action(async (options) => {
-//   const clientConfig = {};
-//   clientConfig['logger'] = new Logger('CLI Logger', LogLevel.WARN, [
-//     new StreamHandler(),
-//   ]);
-//   if (options.verbose) {
-//     clientConfig['logger'].setLevel(LogLevel.DEBUG);
-//   }
-//   clientConfig['nodePath'] = options.nodePath
-//     ? options.nodePath
-//     : utils.getDefaultNodePath();
-
-//   const client = await PolykeyClient.createPolykeyClient(clientConfig);
-//   const vaultMessage = new vaultsPB.Vault();
-//   vaultMessage.setNameOrId(options.vaultName);
-
-//   try {
-//     await client.start({});
-//     const grpcClient = client.grpcClient;
-//     const responseMessage = await binUtils.unaryCallCARL<vaultsPB.Stat>(
-//       client,
-//       attemptUnaryCall(client, grpcClient.vaultsSecretsStat),
```

Comment:
Move this into `bin/secrets`. This should be `pk secrets stat` call that works like an fs stat call.

Reply:
Added.